src/commands/createFunctionApp/functionStacks.ts

```diff
@@ -103,6 +103,26 @@ const linuxFunctionsStacks: string = `{
                 "display": "Node.js",
                 "dependency": null,
                 "majorVersions": [
+                    {
+                        "displayVersion": "14",
+                        "runtimeVersion": "Node|14",
+                        "supportedFunctionsExtensionVersions": [
+                            "~3"
+                        ],
+                        "isDefault": true,
+                        "minorVersions": [],
+                        "applicationInsights": true,
+                        "appSettingsDictionary": {
+                            "FUNCTIONS_WORKER_RUNTIME": "node"
+                        },
+                        "siteConfigPropertiesDictionary": {
+                            "use32BitWorkerProcess": false,
+                            "linuxFxVersion": "Node|14"
+                        },
+                        "isPreview": true,
+                        "isDeprecated": false,
+                        "isHidden": true
```

Comment:
Hiding it?

Reply:
https://github.com/pragnagopa/azure-functions-supported-runtime-stacks/pull/20#issuecomment-703904682